src/main/scala/view/scalaFX/components/BunnyView.scala

```diff
@@ -0,0 +1,106 @@
+package view.scalaFX.components
+
+import model.Bunny
+import model.genome.Alleles
+import scalafx.Includes.{at, double2DurationHelper}
+import scalafx.animation.KeyFrame
+import scalafx.scene.image.{Image, ImageView}
+import view.scalaFX.utilities.Direction
+import view.scalaFX.utilities.Direction._
+import view.utilities.ImageType.{Jumping, Normal}
+import view.utilities.{BunnyImageUtils, ImageType}
+
+import scala.language.postfixOps
+import scala.util.Random
+
+/** Bunny wrapper in order to manage its movement inside of the GUI */
+trait BunnyView {
+  /** Reference to the model bunny entity */
+  val bunny: Bunny
+
+  /** The image of the bunny displayed on the GUI */
+  val imageView: ImageView
+
+  /** The Direction of the bunny jumps */
+  var direction: Direction
+
+  /** The X-Axis position of the bunny */
+  var positionX: Double
+
+  /** The Y-Axis position of the bunny */
+  var positionY: Double
+
+  /** Method that returns the steps to perform a bunny jump
+   * @return      a Seq[KeyFrame] containing the representation of a bunny jump
+   * */
+  def jump(): Seq[KeyFrame]
+}
+
+object BunnyView {
+  val PREFERRED_PANEL_WIDTH = 700
+  val PREFERRED_PANEL_HEIGHT = 200
+  val PANEL_SKY_ZONE = 80
+  val PREFERRED_BUNNY_SIZE = 80
+  val BUNNY_NORMAL_JUMP = 40
```

Comment:
Questa la chiamerei diversamente perchè leggendola la prima volta ho pensato che fosse proprio un intero coniglietto che salta in un certo modo invece che solamente la misura del salto (ex.NORMAL_JUMP_HEIGHT)